### PR TITLE
(publish 8/11) "What's new" Export logs

### PR DESCRIPTION
**Pull request by arenjae**
What's new docs for the new  Export Logs feature

### CODE REVIEW

**Comment by PrashRelic** [comment]
Users were not able to export the logs query results from the Logs and Logs in Content (LiC) UIs. The new Logs Export feature allows export of query results to CSV or JSON format up to the NRDB result limit of 2k. 

I'm the new Logs PMM, please make me the reviewer of it.

**Review by PrashRelic — APPROVED** [review]
Users were not able to export the logs query results from the Logs and Logs in Content (LiC) UIs. The new Logs Export feature allows export of query results to CSV or JSON format up to the NRDB result limit of 2k.

**Review by PrashRelic — APPROVED** [review]
Users were not able to export the logs query results from the Logs and Logs in Content (LiC) UIs. The new Logs Export feature allows export of query results to CSV or JSON format up to the NRDB result limit of 2k.

File: src/content/whats-new/2023/08/whats-new-08-10-export-logs.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: 'Export Logs to CSV or JSON'
+summary: 'Easily download logs from the Logs UI'
+releaseDate: '2023-08-14'
```
Comment (ally-sassman) [inline src/content/whats-new/2023/08/whats-new-08-10-export-logs.md:4]:
```suggestion
releaseDate: '2023-08-11'
```

**Review by ally-sassman — APPROVED** [review]

**Pull request merged**